guide/getting-started/getting-started.md

```diff
@@ -34,70 +34,68 @@ Illustration sources
 
 # Getting started
 
-Welcome to the Bitcoin Design Guide! You’re in for an adventure. This chapter is all about the big concepts that are helpful to be familiar with when creating Bitcoin products. You’ll find a summary of concepts on this page, click through if you would like to dig deeper into a specific topic. The guide isn't a comprehensive overview of all Bitcoin topics, but only the ones you should know as a designer working in this space.
-
 ---
 
 ### [Why Bitcoin is unique]({{ '/guide/getting-started/why-bitcoin-is-unique/' | relative_url }})
 
-The internet has opened an infinite design space for information and communication. Bitcoin adds the new dimension of transferring value that could bring similar change as previous waves of innovation.
+Familiarize yourself with some of the most significant concepts that bitcoin designers need to know.
 
 ---
 
 ### [Why design for Bitcoin]({{ '/guide/getting-started/why-design-for-bitcoin/' | relative_url }})
 
-Bitcoin is fundamentally open and participatory and allows designers and creators to explore and shape experiences for users around the globe.
+Bitcoin is open-source, decentralized, permissionless, fungible, stateless, and non-custodial. If those words don’t mean anything to you, consider starting here.  
 
 ---
 
 ### [Open design]({{ '/guide/getting-started/open-design/' | relative_url }})
 
-Open design is about global collaboration that balances the free exchange of design information, the independence of individual designers and the collective power of collaboration.
+Bitcoin isn’t just changing money—it’s changing how designers collaborate across broad use cases. While open-source development has been around for years, open design is brand new.
```

Comment:
Open design is not a result of Bitcoin. Maybe better to say that collaboration around Bitcoin products requires an open-design approach because Bitcoin itself is open-source based?

Reply:
Attempt to balance things out in https://github.com/BitcoinDesign/Guide/pull/221/commits/7aaf56ee1b2506f5293704ee23824570ab259633 please check

Reply:
I think that’s fine.

Reply:
LGTM with just minor tweaks: `Bitcoin is an open-source protocol. The design process around it should be based on principles of openness and the collective power of collaboration.` There's a `an` missing, and `open-principles` reads funny.